cmd/opm/alpha/diff/cmd.go

```diff
@@ -0,0 +1,101 @@
+package diff
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"github.com/operator-framework/operator-registry/internal/action"
+	"github.com/operator-framework/operator-registry/internal/declcfg"
+	imgreg "github.com/operator-framework/operator-registry/pkg/image/containerdregistry"
+	"github.com/operator-framework/operator-registry/pkg/lib/certs"
+)
+
+const (
+	retryInterval = time.Second * 5
+	timeout       = time.Minute * 1
+)
+
+type diff struct {
+	oldRefs []string
+	newRefs []string
+
+	caFile   string
+	pullTool string
+
+	debug  bool
+	logger *logrus.Entry
+}
+
+func NewCmd() *cobra.Command {
+	a := diff{
+		logger:   logrus.NewEntry(logrus.New()),
+		pullTool: "none",
+	}
+	rootCmd := &cobra.Command{
+		Use: "diff --old { render-target }... --new { render-target }...",
```

Comment:
Do we need `--old` and `--new` flags, or can we just use positional comma-separated args?

Reply:
Either works. I figured, why not make the args obvious with flags. Also if some wrapper script fails to collect the list of old render targets silently, then you'll get a helpful error rather than a heads-only diff:

```sh
local old_targets=""

# Target-lookup logic that fails silently

# Returns an error
opm alpha diff --old ${old_targets} --new reg/catalog:latest
# Generates a heads-only diff, which is likely not what you want
opm alpha diff ${old_targets} reg/catalog:latest
```

I suppose this leads to a discussion of whether heads-only mode is best exposed as-is, but I think by only using `--old/--new` we keep the number of flags low while being descriptive and exposing both modes intuitively.

Reply:
My three main arguments against the flags are:
1) If positional args are good enough for `git diff`, they're probably good enough for `opm diff`
1) In general, non-sqlite based `opm` commands have been shifting to positional args, especially for required arguments, so using positional args would be consistent with the overall CLI design of `opm` going forward.
1) Positional args require more typing.

Reply:
Done.